tests/validator/validator.cpp

```diff
@@ -2348,7 +2420,47 @@ TEST(Validator, unfoundUnitsInEncapsulatedComponents)
     model->linkUnits();
     v->validateModel(model);
 
-    EXPECT_EQ_ERRORS(expectedErrors, v);
+    EXPECT_EQ_ISSUES(expectedIssues, v);
+}
+
+TEST(Validator, mismatchedMultipliersInUnits)
+{
+    const std::vector<std::string> expectedIssues = {
+        "Variable 'v1' in component 'Brobdingnag' has units of 'megametre' and an equivalent variable 'v2' in component 'Lilliput' has compatible but not equivalent units of 'millimetre'. The mismatch is: multiplication factor of 10^9."};
```

Comment:
Add a comma at the end of the issue string, so that ClangFormat can format this better.

Reply:
Done.